doc/dune-files.rst

```diff
@@ -264,70 +264,70 @@ following field to the ``dune-project`` file:
 Dune uses the following global fields to set the metadata for all packages
 defined in the project:
 
-- ``(license <name>)`` - Specifies the license of the project, ideally as an
+- ``(license <name>)`` - specifies the license of the project, ideally as an
   identifier from the `SPDX License List <https://spdx.org/licenses/>`__
 
-- ``(authors <authors>)`` - A list of authors
+- ``(authors <authors>)`` - a list of authors
 
-- ``(maintainers <maintainers>)`` - A list of maintainers
+- ``(maintainers <maintainers>)`` - a list of maintainers
 
 - ``(source <source>)`` - where the source is specified two ways:
   ``(github <user/repo>)`` or ``(uri <uri>)``
 
-- ``(bug_reports <url>)`` - Where to report bugs. This defaults to the GitHub
-  issue tracker if the source is specified as a GitHub repository
+- ``(bug_reports <url>)`` - where to report bugs. This defaults to the GitHub
+  issue tracker, if the source is specified as a GitHub repository.
 
-- ``(homepage <url>)`` - The homepage of the project
+- ``(homepage <url>)`` - the homepage of the project
 
-- ``(documentation <url>)`` - Where the documentation is hosted
+- ``(documentation <url>)`` - where the documentation is hosted
 
-With this fields in, every time dune is called to execute some rules (either via
-``dune build``, ``dune runtest`` or something else), the opam files get
+With these fields, every time one calls Dune to execute some rules (either via
+``dune build``, ``dune runtest``, or something else), the Opam files get
 generated.
 
 Some or all of these fields may be overridden for each package of the project, see
 :ref:`package`.
 
 .. _package:
 
-package
+Package
```

Comment:
Ditto here and other subtitles. They all refer to concrete snippets that must appear in dune files in lower case form (sometimes with underscores).

Reply:
Understood! That’s fine if that’s the way it needs to be for accuracy and clarity. :)